### DISCUSSION TITLE
DPI Analyzer to set 'Tracking Speed' for accurate DPI

### DISCUSSION THREAD

**Discussion by tamedkoala** [linearmouse/linearmouse#647] [Q&A]
I'm confused on how exactly to use this tool ( https://www.mouse-sensitivity.com/dpianalyzer/ ) on a mac to set the 'tracking speed' on linearmouse, given the 'Windows Pointer Speed' data point. If I set tracking to zero, it's obvious to me that this isn't even close as I have to use 2000 DPI to not need a mouse mat that is 10 feet wide. If I set tracking to around .4, this is what I imagine is closer to a default Windows setting, but I'm still confused as to why this DPI tool is recommended if there is no direct correlation between 'tracking speed' and 'windows pointer speed' on the DPI Analyzer.

Is the thought that you would setup the mouse on Windows, do the analyzer, then do it on mac and then adjust 'tracking speed' to achieve the same results you get on Windows? This seems to be the only possible way to accomplish this in my mind, but I'm using a Mac utility for a reason; I don't have Windows. Surely there has to be a default number for 'tracking speed' that closely correlates to the default 'windows pointer speed' that is already known by somebody in this community? Is it actually more complex than this and each person's setup will have a different 'tracking speed' setting depending on their screen size, resolution, etc. instead of this value being universal?

**Comment by hafizirtazaahmed-a11y** [comment]
Using a Mac with Linearmouse and calibrating it with Mouse DPI Analyzers
 can be a bit confusing because Mac’s tracking speed does not directly match Windows Pointer Speed. Mac uses a non-linear curve, so the tracking speed slider in Linearmouse won’t have a one-to-one correlation with Windows settings.

Mouse DPI Analyzers measure how far your cursor moves relative to physical mouse movement. On Mac, they help you see the actual movement per inch and adjust Linearmouse tracking speed until it feels right, approximating the Windows default behavior.

There’s no universal tracking speed because the ideal setting depends on your screen size, resolution, and mouse DPI. Using these analyzers allows you to fine-tune your setup for consistent and precise cursor control.
